tests/unittests/test__init__.py

```diff
@@ -4,12 +4,13 @@
 import os
 import shutil
 import tempfile
+from contextlib import ExitStack
```

Comment:
Good move out of tests.unitests.helpers (ExitStack)  from the stdlib. dunno why we'd ever want to import from a cloud-init helper module. Esp since we didn't modify behavior of ExitStack.